.travis.yml

```diff
@@ -0,0 +1,46 @@
+
+if: (branch = master) OR (tag IS present)  # Add (branch = development) if needed
+language: node_js
+node_js: '12'
+before_install:
+  - yarn install --network-concurrency 1 # workaround https://github.com/yarnpkg/yarn/issues/6312
```

Comment:
once we have deployed to npm components and sdk `--network-concurrency 1` won't be needed.